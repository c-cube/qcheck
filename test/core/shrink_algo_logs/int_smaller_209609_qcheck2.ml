open QCheck2

(* test from issue #59 *)
let ints_smaller_209609 =
  Test.make ~name:"ints < 209609"
    (Gen.small_int_corners()) (Log.shrinks_int @@ fun i -> i < 209609)
;;
try Test.check_exn ints_smaller_209609 with Test.Test_fail _ -> ()
