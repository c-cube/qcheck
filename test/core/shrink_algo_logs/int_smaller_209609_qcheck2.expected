holds 0
holds 1
holds 2
fails 4611686018427387903
